src/components/filters/MeasureValueFilter/Dropdown.tsx

```diff
@@ -67,13 +71,15 @@ class DropdownWrapped extends React.PureComponent<IDropdownProps, IDropdownState
                     onCancel={onCancel}
                     onApply={this.onApply}
                     separators={separators}
+                    displayTreatNullAsZeroOption={displayTreatNullAsZeroOption}
+                    treatNullAsZeroValue={treatNullAsZeroValue}
```

Comment:
as per spec. this should be `treatNullAsZeroDefaultValue` as it is the default value when the filter condition is not present.

Reply:
I see, it true but in another context :) My bad.